lib/rubocop/cop/rspec/factory_bot/factory_class_name.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use string value when setting the class attribute explicitly.
+        #
+        # @example
+        #   # bad
+        #   factory :foo, class: Foo do
+        #   end
+        #
+        #   # good
+        #   factory :foo, class: 'Foo' do
+        #   end
+        class FactoryClassName < Cop
+          MSG = "Pass '%<class_name>s' instead of %<class_name>s."
+
+          def_node_matcher :class_name, <<~PATTERN
+            (send _ :factory _ (hash <(pair (sym :class) $(const ...)) ...>))
+          PATTERN
+
+          def on_send(node)
+            cn = class_name(node)
+
+            return unless cn
```

Comment:
The defined node matcher `class_name` will actually yield the found values to a block, so you could instead do this:
```suggestion
            class_name(node) do |cn|
              add_offense(cn, message: format(MSG, class_name: cn.const_name))
            end
```

Reply:
Ahh great! Thanks.